src/components/Experience.test.ts

```diff
@@ -0,0 +1,170 @@
+import { BattleOutcome } from "battlemovr";
+import { expect } from "chai";
+import { Clock } from "lolex";
+import { IListMenuProgress, MenuGraphr } from "menugraphr";
+import { Children } from "react";
+import { FullScreenPokemon } from "../";
+import { stubBlankGame } from "../fakes.test";
+import { IBattleInfo, IBattleTeam, IPartialBattleOptions, IPokemon } from "./Battles";
+import { IPlayer } from "./Things";
+
+const skipBattle = (clock: Clock, fsp: FullScreenPokemon, player: IPlayer): void => {
+    fsp.inputs.keyDownA(player);
+    clock.tick(2000);
```

Comment:
The branch I showed you was just a vague hackery, you shouldn't need all these big `clock.tick`s. See which ones aren't necessary (most shouldn't be) and remove them.

Reply:
Without these ticks or without lower values the menus get stuck and the battle doesn't complete.  Reducing some of the ticks makes the non level up battle test work but the level up one fail since it has an extra menu to process. Removing the clock.tick calls will cause the battle to not complete however and the tests to fail.